example/gardener-local/kind/cluster-skaffold.yaml

```
@@ -38,6 +43,10 @@ nodes:
     apiVersion: kubelet.config.k8s.io/v1beta1
     kind: KubeletConfiguration
     maxPods: 500
+containerdConfigPatches:
+- |-
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:5001"]
+    endpoint = ["http://10.2.0.11:5001"]
```

Should we use the same endpoint here as in kind2?
```suggestion
    endpoint = ["http://gardener-local-control-plane:30501"]
```